harness/determined/cli/workspace.py

```diff
@@ -182,13 +232,19 @@ def create_workspace(args: argparse.Namespace) -> None:
         defaultComputePool=args.default_compute_pool,
         defaultAuxPool=args.default_aux_pool,
     )
+    if args.namespace:
+        cluster_name = args.cluster_name or ""
```

Comment:
The behavior of setting cluster_name to "" is a little unclear to me. If it's meant for users with single RM to create a workspace using `det w create myworkspace --n mynamespace`, can we add a comment and include this behavior in `help` attribute of `cli.Arg("--cluster-name")`The behavior of setting cluster_name to "" is a little unclear to me. If it's meant for users with single RM to create a workspace using `det w create myworkspace --n mynamespace`, can we add a comment and include this behavior in `help` attribute of `cli.Arg("--cluster-name")`

Reply:
I added that line because plugging `args.cluster_name` into the `content.clusterNamespacePairs` dictionary plugs in the word `null` when the cluster name is not specified, rather than an empty string (and on the backend we match the empty string with empty `cluster-name` args). I'll comment that!
And good point about the arg being optional for single RM specifically, I'll add that in the `help` attribute!